prow/cmd/pipeline/controller.go

```diff
@@ -557,6 +557,12 @@ func reconcile(c reconciler, key string) error {
 		if err != nil {
 			return fmt.Errorf("no pipelinerun found with name %s: %v", name, err)
 		}
+
+		// Don't reconcile any PipelineRuns that don't have the created-by-prow label set to true.
+		switch v, ok := p.Labels[kube.CreatedByProw]; {
```

Comment:
why not using:
```Go
if  v, ok := p.Labels[kube.CreatedByProw]; !ok && v != "true" {
   return nil
}

It seems simpler.
```

Reply:
Will do - I copy-pasted.

Reply:
Actually, wouldn't that be `!ok || v != "true"`?

Reply:
yes! Sorry

Reply:
`&&` does actually work for this specific case, since if `ok` is false, `v` is definitely not gonna equal `"true"`, so I changed it to that...but now I realize that won't work if for some reason someone has another value in `created-by-prow`, so switching it to `||`. heh.